cmd/ccboc/main.go

```diff
@@ -121,6 +121,88 @@ func getCalculationID(c *cli.Context) error {
 	return nil
 }
 
+func getCalculationResult(c *cli.Context) error {
+	teff := c.Float64("teff")
+	logG := c.Float64("logG")
+	path := c.String("path")
+
+	resp, err := http.Get(globalConfig.APIURL + "/calculations/results?teff=" + fmt.Sprintf("%0.1f", teff) + "&logg=" + fmt.Sprintf("%0.1f", logG))
+	if err != nil {
+		logrus.WithError(err).Fatal("Couldn't retrieve the data from headers.")
+	}
+
+	key := resp.Header.Get("filename")
+
+	var defaultPath string
+
+	if path != "" {
+		defaultPath = filepath.Join(path, filepath.Base(key))
+	} else {
+		defaultPath = filepath.Join("~/.config/ccboc/", filepath.Base("text.txt"))
+	}
+
+	if _, err := os.Stat(defaultPath); os.IsNotExist(err) {
+		logrus.WithError(err).Fatal("The desired file path does not exist.")
```

Comment:
Let us have a better error message here. Also, I think it will be better to just print a warning here and then download the file to the default path.